### PR TITLE
Define project's objective

### PR DESCRIPTION
**Pull request by McSinyx**
I should have written the introduction but my brain refuses to produce literature today.

Hopefully resolve GH-16 ([sorry 4 the wait](https://en.wikipedia.org/wiki/Sorry_4_the_Wait) though).

### CODE REVIEW

**Comment by Huy-Ngo** [comment]
Can the user define their own Kanban columns? I thought we would allow that with the 3 columns as default.

File: docs/source/obj.rst [hunk 1]
```diff
@@ -1,2 +1,39 @@
...
+
+The Collaboration Platform
+--------------------------
+
+At the lowest level, discussion would be facilitated through comment threads,
+organized into logical tasks.  The tasks would be categorized by stages
+such as *to-do*, *doing* and *done*.  These stages would then be visualized
+as columns in a `Kanban board`_ to aid managing and prioritizing tasks.
+
+The Academic Itergration
```
Comment (Huy-Ngo) [inline docs/source/obj.rst:18]:
```suggestion
The Academic Integration
```

**Review by Huy-Ngo — CHANGES_REQUESTED** [review]
There's a spelling mistake, as always.

File: docs/source/obj.rst (hunk 1 above)
Comment (McSinyx) [reply docs/source/obj.rst:18]: Huh I should use word completion for every word now :smile:

Comment (Huy-Ngo) [reply docs/source/obj.rst]: You can try something like `ispell` but it nitpicks valid words like "workflow" or "Kanban" or "http".

**Review by Huy-Ngo — APPROVED** [review]

File: docs/source/obj.rst (hunk 1 above)
Comment (McSinyx) [reply docs/source/obj.rst]: I can use the builtin spell, but never rmb to turn it on.  Maybe I should make it the default for markup languages :shrug:

**Review by DDHLong — APPROVED** [review]

**Comment by McSinyx** [comment]
Thanks!

**Pull request merged**